.eslintrc.js

```diff
@@ -17,21 +16,20 @@ module.exports = {
       "flowtype"
     ],
     "rules": {
-        "indent": [
-            "error",
-            2
-        ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ]
+      "no-magic-numbers": [
+        "off"
+      ],
+      "quotes": [
+        "error",
+        "single"
+      ],
+      "comma-dangle": [
+        "error",
+        "always-multiline"
```

Comment:
Can you clarify what you mean by breaking code on modern browsers if pasting non-transpiled code? I don't think I understand that use case. Pasting almost anything from here directly into a browser would break since its typed with flow.

It's also worth noting that [react-game-kit](https://github.com/FormidableLabs/react-game-kit/blob/master/.eslintrc#L10) and [react-music](https://github.com/FormidableLabs/react-music/blob/master/.eslintrc#L8) both use `"always-multiline"`.